modules/accredited_representative_portal/db/migrate/20241212000000_create_ar_power_of_attorney_request_resolutions.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateArPowerOfAttorneyRequestResolutions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ar_power_of_attorney_request_resolutions, id: :uuid do |t|
+      t.string 'resolving_type', null: false
+      t.uuid 'resolving_id', null: false
+      t.text 'reason_ciphertext'
+      t.datetime 'created_at', null: false
+    end
+
+    create_table :ar_power_of_attorney_request_replacements, id: :uuid
+
+    create_table :ar_power_of_attorney_request_withdrawals, id: :uuid
```

Comment:
I think we can omit replacements and withdrawals for the time being.

We don't really know what product thinks about submitting a new POA request when another one already exists in our system. Until we broach this question with product, we can have no logic here and just allow a new POA request creation.

Similarly for withdrawals, I have not heard any discussion of allowing for the claimant to withdraw a given POA request.